docs/guide/built-in-rules-callback.md

```diff
@@ -87,10 +86,10 @@ new Callback(
 > **Note:** Processing untrusted user input with `yaml_parse()` can be dangerous with certain settings. Please refer to
 > [yaml_parse docs] for more details. 
 
-### Usage of validation context for validating multiple attributes depending on each other
+### Using a validation context to validate interdependent attributes
```

Comment:
`interdependent` is significantly less used and can be harder to understand, especially for non-native speakers.